### PR TITLE
fix: regexp changed to catch all urls and u/e-numbers

### PR DESCRIPTION
**Pull request by grigoriev**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -20,13 +20,12 @@ repos:
...
         name: Sensitive data leak - URLs
-        entry: sbb\.ch\b
+        entry: (?<!polarion-opensource@)(?<!www\.)sbb\.ch
         language: pygrep
         types: [text]
-        exclude: \b(www\.|polarion-opensource@)sbb\.ch\b
       - id: sensitive-data-leak-ue-numbers
         name: Sensitive data leak - UE numbers
-        entry: ([uUeE]{1,2})\d{5,6}
+        entry: \s([uUeE]{1,2})\d{5,6}\s
```
Comment (ariwk) [inline .pre-commit-config.yaml:28]: it does not recognise 
u12345
u123446@sbb.ch
<u123456>

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -20,13 +20,12 @@ repos:
     hooks:
       - id: sensitive-data-leak-urls
         name: Sensitive data leak - URLs
-        entry: sbb\.ch\b
+        entry: (?<!polarion-opensource@)(?<!www\.)sbb\.ch
```
Comment (ariwk) [inline .pre-commit-config.yaml:23]: it does not recognise:
- u123446@sbb.ch
- <XXXXX-opensourceq@sbb.ch>

**Review by ariwk — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (grigoriev) [reply .pre-commit-config.yaml:28]: Fixed

File: .pre-commit-config.yaml (hunk 2 above)
Comment (ariwk) [reply .pre-commit-config.yaml:23]: and this one? XXXX-polarion-opensource@sbb.ch

Comment (grigoriev) [reply .pre-commit-config.yaml:23]: No, but I'm not sure that makes sense. Are you sure you want to cover this case as well?

Comment (ariwk) [reply .pre-commit-config.yaml:23]: no

**Review by ariwk — APPROVED** [review]

**Pull request merged**